docs/_adapters/cpx.md

```diff
@@ -10,6 +10,23 @@ github_link: https://github.com/meshery/meshery-cpx
 image: /assets/img/service-meshes/citrix.svg
 ---
 
+{% assign sorted_tests_group = site.compatibility | group_by: "meshery-component" %}
+{% for group in sorted_tests_group %}
+      {% if group.name == "meshery-cpx" %}
+        {% assign items = group.items | sort: "meshery-component-version" | reverse %}
+        {% for item in items %}
+          {% if item.meshery-component-version != "edge" %}
+            {% if item.overall-status == "passing" %}
+              {% assign adapter_version_dynamic = item.meshery-component-version %}
+              {% break %}
+            {% elsif item.overall-status == "failing" %}
+              {% continue %}
+            {% endif %}
+          {% endif %}
+        {% endfor %} 
+      {% endif %}
+{% endfor %}
+
```

Comment:
Isn't CPX deprecated as of now? // @leecalcote

Reply:
> Isn't CPX deprecated as of now? // @leecalcote

@alphaX86, yes, sir, it is.

Reply:
@ayushthe1 note the above ☝️ you have a green light of removing the CPX adapter page as well